ldm/invoke/conditioning.py

```diff
@@ -57,10 +57,18 @@ def get_uc_and_c_and_ec(prompt_string, model, log_tokens=False, skip_normalize_l
     positive_prompt_string, negative_prompt_string = split_prompt_to_positive_and_negative(prompt_string)
     legacy_blend = try_parse_legacy_blend(positive_prompt_string, skip_normalize_legacy_blend)
     positive_prompt: FlattenedPrompt|Blend
+    lora_conditions = None
     if legacy_blend is not None:
         positive_prompt = legacy_blend
     else:
         positive_prompt = Compel.parse_prompt_string(positive_prompt_string)
+        should_use_lora_manager = True
+        if model.peft_manager:
+            should_use_lora_manager = model.peft_manager.should_use(positive_prompt.lora_weights)
+            if not should_use_lora_manager:
+                model.peft_manager.set_loras(positive_prompt.lora_weights)
```

Comment:
Unsure how beneficial this is, has anyone even tested peft formatted LoRA?

Reply:
Indeed not. Where do you even find a peft formatted LoRA to test with?

Reply:
I've never heard of anyone training them outside of random tests. Unfortunately, the example linked from huggingface/peft [is disabled](https://huggingface.co/spaces/smangrul/peft-lora-sd-dreambooth), and I cannot find any SD PEFT LoRA anywhere.